plugins/available/docker.plugin.bash

```diff
@@ -33,14 +34,17 @@ function docker-remove-images() {
 	about 'attempt to remove images with supplied tags or all if no tags are supplied'
 	group 'docker'
 	if [ -z "$1" ]; then
+		# shellcheck disable=SC2046
 		docker rmi $(docker images -q)
 	else
 		DOCKER_IMAGES=""
-		for IMAGE_ID in $@; do DOCKER_IMAGES="$DOCKER_IMAGES\|$IMAGE_ID"; done
+		for IMAGE_ID in "$@"; do DOCKER_IMAGES="$DOCKER_IMAGES\|$IMAGE_ID"; done
```

Comment:
```suggestion
		for IMAGE_ID; do DOCKER_IMAGES="$DOCKER_IMAGES\|$IMAGE_ID"; done
```

Reply:
not liking this. I didn't even know about this syntax option. I prefer to keep it explicit and clear.

Reply:
Hmm, I feel it strange, but OK.